### ISSUE TITLE
installation instructions outdated in readme

### ISSUE THREAD

**Issue by jcheong0428** [nipy/heudiconv#112]
curl -O https://raw.githubusercontent.com/nipy/heudiconv/master/bin/heudiconv
not working, file does not seem to be in repo?

**Comment by jcheong0428** [comment]
looks like /bin/heudiconv 
was deleted in the latest merge.

**Comment by yarikoptic** [comment]
Thanks! Readme should be adjusted to show that it is a proper python module now and needs pip install command. I think you should be able to pip install heudiconv which would give you the last released version

**Comment by jcheong0428** [comment]
sorry still not working with following error 

`Traceback (most recent call last):
  File "/Users/jinhyuncheong/anaconda2/bin/heudiconv", line 11, in <module>
    load_entry_point('heudiconv==0.4', 'console_scripts', 'heudiconv')()
  File "/Users/jinhyuncheong/anaconda2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Users/jinhyuncheong/anaconda2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2755, in load_entry_point
    return ep.load()
  File "/Users/jinhyuncheong/anaconda2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2409, in load
    return self.resolve()
  File "/Users/jinhyuncheong/anaconda2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2415, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named heudiconv.cli.run`

**Comment by yarikoptic** [comment]
Ha... apparently we never uploaded it to pypi... shame... did now: https://pypi.python.org/pypi/heudiconv
(conda env) pip installed it, realized that dcmstack isn't on pip, but was available via conda, so conda installed it.   Should work now

We will come up with a conda recipe some time soon https://github.com/conda-forge/staged-recipes/issues/4358

**Comment by satra** [comment]
@yarikoptic - a special version of dcmstack is required - specifically from @mvdoc - the conda forge release may not work - has anyone run the tests with that release?

btw, the intent was not to release before all things were taken care of. so now we should fix those.

**Comment by yarikoptic** [comment]
oh, I forgot that dcmstack is in such an abandoned shape. Let's see if we could bring Brendan's attention to kick out a fixed up release... patching a fork (and even that PR isn't merged) is kinda a next level of being abandoned

**Comment by yarikoptic** [comment]
email is out
and `ha ha` to `not to release before ...` ;)

**Comment by mgxd** [comment]
@jcheong0428 this was an installation issue that was fixed in #117 , you should now be able to install with
```
pip install -U https://github.com/nipy/heudiconv/archive/master.zip
```

@yarikoptic we should think of releasing `0.4.1` or `0.5` soon

**Comment by yarikoptic** [comment]
This will be more like 0.100 or so since someone was to active @mgxd  ;-) please contribute a summary of changes to https://github.com/nipy/heudiconv/pull/118
I will meanwhile also update NeuroDebian package to be at least of 0.4 state
